From b184152761d8a4dbf73adcffaddac85709949c2f Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 15 Feb 2009 16:16:51 -0500
Subject: [PATCH] daemon: use sys/ioctl.h for ioctl()

Including stropts.h for the ioctl() prototype is weird and breaks some
systems that do not implement stropts.h.  Use sys/ioctl.h like normal.

URL: http://bugs.gentoo.org/222099
Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Reported-by: Sergey Dryabzhinsky <sergey.dryabzhinsky@gmail.com>
---
 src/daemon/open_console.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/daemon/open_console.c b/src/daemon/open_console.c
index 27b62e2..98297c9 100644
--- a/src/daemon/open_console.c
+++ b/src/daemon/open_console.c
@@ -21,7 +21,7 @@
 
 #include <fcntl.h>                  /* open and co.      */
 #include <sys/stat.h>               /* stat()            */
-#include <stropts.h>                /* ioctl             */
+#include <sys/ioctl.h>              /* ioctl             */
 
 /* Linux specific (to be outsourced in gpm2 */
 #include <linux/serial.h>           /* for serial console check */
-- 
1.6.1.3

